frontend/src/features/shared/patient/detail/PatientDetailSection.tsx

```diff
@@ -0,0 +1,143 @@
+import { useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardTitle,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+} from '@freemed-kit/ui';
+import { ReceiptFormValues } from '@/features/shared/receipt/useReceiptForm';
+
+const PatientDetailSection = () => {
+  const form = useFormContext<ReceiptFormValues>();
+  const { watch, setValue } = form;
+  const height = watch('height');
+  const weight = watch('weight');
+
+  useEffect(() => {
+    if (height && weight && height >= 1 && weight >= 1) {
+      const heightInMeters = height / 100;
+      const bmi = weight / (heightInMeters * heightInMeters);
+      setValue('bmi', parseFloat(bmi.toFixed(2)));
+    }
```

Comment:
```suggestion
    } else {
      setValue('bmi', null);
    } 
```
신장, 체중 입력하다가 둘 중 하나라도 지우면 BMI도 초기화해야 합니다!